sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- [[#6087]](https://github.com/Azure/azure-sdk-for-cpp/pull/6087) Set version property for the compiled SDK binary files. (A community contribution, courtesy of _[chewi](https://github.com/chewi)_)
```

Comment:
I wonder if it's appropriate to put this under "new features", since it doesn't impact callers. It seems better fit for "other changes", since it's more of a infra/packaging change.

Reply:
I wasn't sure myself, I'll move it to Other Changes then.
I think, if you have a setup with symlinks like described in https://github.com/Azure/azure-sdk-for-cpp/pull/6087, this could feel more like a feature, however if you look at "New Feaures" strongly from the "New API/New Behabior" perspective, this should fall under Other Changes.